src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -52,12 +59,19 @@ const convertInclude = async ({ href, type, ...rest }) => {
     'smallprox/include': 'vj',
   };
 
+  const { href, type, ...rest } = includeBlock;
```

Comment:
Pulled this line down here so that the error log for `INCLUDE_MISSING_URL` could include the whole includeBlock, to help us debug issues relating to a broken include.